src/config/tokens/core/_core-fonts.scss

```diff
@@ -26,6 +26,7 @@ $font-weight-bold: 500;
 
 // - Font family
 $font-family: "Roboto", "Rubik", "Noto Kufi Arabic", "Noto Sans JP", sans-serif;
+$h1-font-family: "Poppins", "Rubik", "Noto Kufi Arabic", "Noto Sans JP", sans-serif;
```

Comment:
@3dyonic lets add Poppins with Roboto, so it won't break until we will add it to the monolith

Reply:
done